### ISSUE TITLE
Save images locally

### ISSUE THREAD

**Issue by sangjunp4rk** [willis1298/prj-rev-bwfs-dasmoto#1]
Try to save your images locally instead of adding a link. Saving images adds to the stability of your website because your website is not reliant on the link to be valid all the time. So next time, save all the images under resources/images/

https://github.com/willis1298/prj-rev-bwfs-dasmoto/blob/master/Dasmotos/index.html#L14

**Comment by willis1298** [comment]
Yes, at first, I realized that's better, so I created the jpeg files in my resources folder to link to.  Then I used the construction: <img alt="Frames" src="Frames.jpg"> for each image, but it didn't work.  How is that construction wrong?

**Comment by willis1298** [comment]
Ok, interesting.  That would be , for example, <img alt="image name" src="image name.jpg"

**Comment by willis1298** [comment]
Nope, sorry.  Again: <img alt="image name" src="image name.jpg">

**Comment by willis1298** [comment]
Ok, my code in the comment is live.  You can inspect it.
